.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
       - image: circleci/golang:1.12.9-browsers
       - image: 0xorg/ganache-cli:latest
         environment:
-            VERSION: 4.3.0
+            VERSION: 4.3.3
```

Comment:
The V2 & V3 Ganache snapshots did not have ERC1155 Mintable tokens deployed on them so I had to add it.